public/apps/configuration/configuration-app.tsx

```diff
@@ -21,14 +21,22 @@ import { I18nProvider } from '@osd/i18n/react';
 import { AppMountParameters, CoreStart } from '../../../../../src/core/public';
 import { SecurityPluginStartDependencies, ClientConfigType } from '../../types';
 import { AppRouter } from './app-router';
+import { DataSourceManagementPluginSetup } from '../../../../../src/plugins/data_source_management/public';
 
 export function renderApp(
   coreStart: CoreStart,
-  navigation: SecurityPluginStartDependencies,
+  securityPluginStartDeps: SecurityPluginStartDependencies,
```

Comment:
nit: can we rename this to depsStart. I see this naming used across OSD core in a few plugins.

Can we also change every instance of where `navigation` appears to `depsStart`?

Reply:
Yes - I thought navigation was misleading and named it to securityPluginStartDeps and changed all the references. I can change it to depsStart, to follow the convention.